content/migrations/0006_merge.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this makes it so theres two 0006 migrations. Seems like it'll break migrations?